<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#Form
{
	background-color: rgb(213, 213, 213);
}


QWidget
{
	background-color: rgb(239, 239, 239);
}


QPushButton 
{
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 rgba(234, 234, 234, 255), 
stop:0.5 rgba(249, 249, 249, 255),
stop:1 rgba(234, 234, 234, 255));
	border-style: outset;
    border-width: 2px;
    border-color: rgba(234, 234, 234, 255)
 }

QPushButton#pushButton_register
 {
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 rgba(234, 234, 234, 255), 
stop:0.5 rgba(249, 249, 249, 255),
stop:1 rgba(234, 234, 234, 255));
	border-style: outset;
    border-width: 2px;
    border-color: rgba(128, 128, 128, 255)
 }

 QPushButton:pressed 
{
	background-color: rgb(221, 221, 221);
 }

 QPushButton:hover {
	 background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 rgba(229, 229, 229, 255), 
stop:0.5 rgba(221, 221, 221, 255),
stop:1 rgba(229, 229, 229, 221));
    border-color: rgba(221, 128, 32, 255)
 }


QLineEdit
 {
	background-color:  rgba(255, 255, 255, 255);
	border-style: outset;
    border-width: 2px;
    border-color: rgba(234, 234, 234, 255)
 }

QLabel
{
    padding: 2px;
}
</string>
  </property>
  <widget class="QLabel" name="label_01_username">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_01_username">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_register">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>480</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_00_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_02_password">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_05_firstname">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_03_repassword">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Re-Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_08_address2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Address2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_09_province">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Province</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_07_address1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Address1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_06_lastname">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11_zipcode">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zip Code</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10_country">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Country</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10_country_detail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(Ex: Thailand)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_07_address1_detail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>up to 28 characters (Ex: 99 Abc Rd.)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_06_lastname_detail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>up to 28 characters (Ex: Troyard)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_05_firstname_detail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>up to 28 characters (Ex: Smith)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_09_province_detail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(Ex: Bangkok)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_08_address2_detail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>up to 28 characters (optional)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_03_repassword_detail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Re-enter Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_02_password_detail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4-12 characters (Ex: 004#Smith)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11_zipcode_detail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(Ex: 10990)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_01_username_detail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4-12 characters (Ex: smith)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_02_password">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_05_firstname">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_03_repassword">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_08_address2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_07_address1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_06_lastname">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_09_province">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10_country">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_11_zipcode">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_04_email">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_04_email">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>E-mail</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_04_email_detail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(Ex: uMail@uAuction.com)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_12_phonenumber">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_12_phonenumber_detail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(Ex: 0812345678)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12_phonenumber">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Phone</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>label_02_password</zorder>
  <zorder>label_01_username</zorder>
  <zorder>lineEdit_01_username</zorder>
  <zorder>pushButton_register</zorder>
  <zorder>label_05_firstname</zorder>
  <zorder>label_03_repassword</zorder>
  <zorder>label_08_address2</zorder>
  <zorder>label_09_province</zorder>
  <zorder>label_07_address1</zorder>
  <zorder>label_06_lastname</zorder>
  <zorder>label_11_zipcode</zorder>
  <zorder>label_10_country</zorder>
  <zorder>label_10_country_detail</zorder>
  <zorder>label_07_address1_detail</zorder>
  <zorder>label_06_lastname_detail</zorder>
  <zorder>label_05_firstname_detail</zorder>
  <zorder>label_09_province_detail</zorder>
  <zorder>label_08_address2_detail</zorder>
  <zorder>label_03_repassword_detail</zorder>
  <zorder>label_02_password_detail</zorder>
  <zorder>label_11_zipcode_detail</zorder>
  <zorder>label_01_username_detail</zorder>
  <zorder>lineEdit_02_password</zorder>
  <zorder>lineEdit_05_firstname</zorder>
  <zorder>lineEdit_03_repassword</zorder>
  <zorder>lineEdit_08_address2</zorder>
  <zorder>lineEdit_07_address1</zorder>
  <zorder>lineEdit_06_lastname</zorder>
  <zorder>lineEdit_09_province</zorder>
  <zorder>lineEdit_10_country</zorder>
  <zorder>lineEdit_11_zipcode</zorder>
  <zorder>label_00_title</zorder>
  <zorder>lineEdit_04_email</zorder>
  <zorder>label_04_email</zorder>
  <zorder>label_04_email_detail</zorder>
  <zorder>lineEdit_12_phonenumber</zorder>
  <zorder>label_12_phonenumber_detail</zorder>
  <zorder>label_12_phonenumber</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEdit_01_username</tabstop>
  <tabstop>lineEdit_02_password</tabstop>
  <tabstop>lineEdit_03_repassword</tabstop>
  <tabstop>lineEdit_04_email</tabstop>
  <tabstop>lineEdit_05_firstname</tabstop>
  <tabstop>lineEdit_06_lastname</tabstop>
  <tabstop>lineEdit_07_address1</tabstop>
  <tabstop>lineEdit_08_address2</tabstop>
  <tabstop>lineEdit_09_province</tabstop>
  <tabstop>lineEdit_10_country</tabstop>
  <tabstop>lineEdit_11_zipcode</tabstop>
  <tabstop>lineEdit_12_phonenumber</tabstop>
  <tabstop>pushButton_register</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
